config/liota.conf

```diff
@@ -17,6 +17,8 @@ enable_reboot_getprop = False
 iotcc_load_retry = 3
 # Configurable queue timeout in order to wait for the response messages from IOTCC
 iotcc_response_timeout = 300
+# System Properties List to be set during registration of Edge System/Devices
+system_properties= {}
```

Comment:
We need to divide this into two groups or multiple files. We don't want users to set system_properties, but we want them to change iotcc_response_timeout.

Reply:
This can be taken up later in the current approach we can use the conf file to set the properties as it is required only by Airwatch which integrates with Liota as user.

Reply:
Sure, we can take this up later. Please raise a bug for this. For the end users, we might end up telling them to change iotcc_response_timeout, collect_thread_pool_size, crl_path etc. but that should not lead them to try out changing other properties like system_properties, which are only for airwatch agent.